mon-pix/app/routes/organizations.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import RSVP from 'rsvp';
+
+export default class OrganizationsRoute extends Route {
+  @service store;
+
+  async model(params) {
+    return RSVP.hash({
+      organizationToJoin: this.store.queryRecord('organization-to-join', { code: params.code }),
+      campaign: this.store.queryRecord('campaign', { filter: { code: params.code } }),
+    });
```

Comment:
Pas besoin de RSVP.hash, deux await et un objet en retour font très bien l'affaire 
```suggestion
   const organizationToJoin = await this.store.queryRecord('organization-to-join', { code: params.code }),
   const campaign = await  this.store.queryRecord('campaign', { filter: { code: params.code } }),
   return { organizationToJoin, campaign };
```